language/move-prover/tests/sources/functional/cross_module_invariant.move

```diff
@@ -9,12 +9,15 @@ module TestCrossModuleInv {
     }
 
     spec module {
+        // TODO: this test has been deactivated because of non-determinism in the generated prover diagnosis
+        //   The test should be rewritten to not depend on libra framework. Functional tests should be standalone.
```

Comment:
Yes I am aware of this issue. Rain noticed it in her [PR](https://github.com/diem/diem/pull/7161/files#diff-6f883142cd37eba1fe67939369af66f7ff40bd6d7f357f26538a2e83c4216800R18) and asked me how to fix it. Will work on rewriting it.